tests/__init__.py

```diff
@@ -99,6 +99,11 @@ class FoobarParams:
     params: FoobarParams = FoobarParams()
 
 
+@dataclass
+class TypedFields:
+    bar: float = 123.456
+
+
```

Comment:
As an alternative to creating a type here, you can use:
```
DictConfig({"bar": FloatNode(123.456)})
```

Reply:
Ok, I made the change in https://github.com/omry/omegaconf/pull/744/commits/f1b441d0a9cf3fc4d148200a1f379601bf1b76cc.